### PR TITLE
chore(clickhouse): Capture final SQL in Sentry errors, again

### PR DESCRIPTION
**Pull request by Twixes**
Reverts PostHog/posthog#21479.

### CODE REVIEW

**Comment by Twixes** [comment]
In fairness, I only wanted to upgrade `clickhouse-driver` to unblock the upgrade to Py 3.11. To do that, we only need `clickhouse-driver==0.2.6`, and not the problematic `clickhouse-driver==0.2.7`. So 0.2.6 it is.

File: .github/workflows/ci-backend.yml [hunk 1]
```diff
@@ -197,22 +197,29 @@ jobs:
...
+              run: |
                   python manage.py migrate
 
-            - uses: actions/checkout@v3
+            # Now we can consider this PR's migrations
+
+            - name: Checkout this PR
+              uses: actions/checkout@v3
+
+            - name: Install python dependencies for this PR
```
Comment (neilkakkar) [inline .github/workflows/ci-backend.yml:220]: this also needs to run this PR's migrations, I don't think check migrations below does that

**Review by tomasfarias — APPROVED** [review]
lgtm with the caveat of open conversation being  resolved.

File: .github/workflows/ci-backend.yml (hunk 1 above)
Comment (Twixes) [reply .github/workflows/ci-backend.yml:220]: Weirdly, it should, yet it looks like it never did. 🤔 Let's add that, sure

**Pull request merged**